pkg/pubsub/node_data.go

```diff
@@ -220,10 +220,20 @@ func (n *NodeData) GetAccumulatedUptime() time.Duration {
 // Otherwise, it uses the time since the last joined event.
 func (n *NodeData) UpdateAccumulatedUptime() {
 	if n.Activity == ActivityLeft {
-		n.AccumulatedUptime += time.Since(time.Unix(n.LastLeftUnix, 0))
```

Comment:
welp - I fail to find where this function is actually called: 

https://github.com/search?q=repo%3Amasa-finance%2Fmasa-oracle%20UpdateAccumulatedUptime&type=code

Reply:
```
$ git grep "AccumulatedUptime"
pkg/pubsub/node_data.go:// UpdateAccumulatedUptime updates the accumulated uptime of the node to account for any
pkg/pubsub/node_data.go:func (n *NodeData) UpdateAccumulatedUptime() {
```

Reply:
Seems https://github.com/masa-finance/masa-oracle/blob/4744de318e7c76f28307e75e99e5f902c9c993dc/pkg/pubsub/node_data.go#L113 is the one we want to touch? maybe this should be dropped as it causes confusion

Reply:
maybe @jdutchak or @restevens402 knows where this is actually consumed?